src/test/integration/requirements.txt

```diff
@@ -4,4 +4,4 @@ idna==2.6
 phonenumbers==8.9.2
 Pillow==5.0.0
```

Comment:
This version isn't supported by Python 3.7: https://pillow.readthedocs.io/en/latest/installation.html#notes The installation will fail with pip. We should probably either update the Docker python version or update this package version

Reply:
Good catch, the python 3 docker image in jenkins vm is 3.6 so I didn't run into this issue while testing. Will fix.